app/main.js

```diff
@@ -35,8 +41,38 @@ function createWindow () {
   mainWindow.on("closed", function () {
     mainWindow = null
   })
+
+  ipcMain.on("initialized", () => {
+    if (deepLinkURLOnReady != null) {
```

Comment:
Is it worth resetting this `deepLinkURLOnReady` global after it's been passed to the renderer process, to prevent accidentally calling `loadPopulatePage` multiple times?